config/common/errors.go

```diff
@@ -69,6 +69,7 @@ var (
 	ErrUserFieldSupport       = errors.New("fields other than \"name\" and \"ssh_authorized_keys\" are not supported in this spec version")
 	ErrUserNameSupport        = errors.New("users other than \"core\" are not supported in this spec version")
 	ErrKernelArgumentSupport  = errors.New("this field cannot be used for kernel arguments in this spec version; use openshift.kernel_arguments instead")
+	ErrClevisSupport          = errors.New("clevis is not supported in this spec version")
```

Comment:
Let's add a new block for this, maybe `// Storage`.  This error doesn't relate to MachineConfigs.